hack/istio/multicluster/install_ambient_multicluster.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+# Helper function to install ambient multicluster for a single cluster
+install_ambient_on_cluster() {
+  local context="${1}"
+  local user="${2}"
+  local pass="${3}"
+  local cluster_name="${4}"
+  local network="${5}"
+  
+  echo "==== INSTALL ISTIO ON CLUSTER [${cluster_name}] - ${context}"
+  switch_cluster "${context}" "${user}" "${pass}"
+
+  echo "Installing Istio with ambient profile on cluster: ${cluster_name}, network: ${network} and meshID: ${MESH_ID}"
+
+  # Label the namespace
+  kubectl --context="${context}" label namespace istio-system topology.istio.io/network=${network}
+
+  # Create the IstioOperator configuration for ambient multi-cluster
+  local config_yaml=$(mktemp)
+  cat <<EOF > "${config_yaml}"
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  profile: ambient
+  components:
+    pilot:
+      k8s:
+        env:
+          - name: AMBIENT_ENABLE_MULTI_NETWORK
+            value: "true"
+  values:
+    global:
+      meshID: ${MESH_ID}
+      multiCluster:
+        clusterName: ${cluster_name}
+      network: ${network}
```

Comment:
Does this work with Sail as well? Since we moved the other installs to Sail it would be good to be consistent.

Reply:
Ref. https://github.com/istio-ecosystem/sail-operator/issues/1210